lib/cartopy/tests/mpl/test_gridliner.py

```diff
@@ -126,7 +126,7 @@ def test_gridliner_specified_lines():
 
 # The tolerance on these tests are particularly high because of the high number
 # of text objects. A new testing strategy is needed for this kind of test.
-if MPL_VERSION < "3":
+if MPL_VERSION.major < 3:
```

Comment:
We have a minimum dependency of MPL 3.2, so I guess this can just be removed? There are quite a few checks in the tests that we should clean up other than this as well, so this can wait for another PR.

Reply:
Do we? The code doesn't seem to say that anywhere.

Reply:
According to our CI config, it's matplotlib 3.1:

https://github.com/SciTools/cartopy/blob/c7af419dd452450f52e9987e00e764f6e18c8e42/.github/workflows/ci-testing.yml#L26-L33

😆 The docs currently say:
https://github.com/SciTools/cartopy/blob/c7af419dd452450f52e9987e00e764f6e18c8e42/INSTALL#L103-L105

Reply:
Oh lovely, `requirements/plotting.txt` says 1.5.1 as well.

Reply:
#1852